dao/src/integ-test/java/uk/gov/moj/sdt/dao/IndividualRequestDaoTest.java

```diff
@@ -161,10 +166,7 @@ public void testGetIndividualRequestPastRetention() {
         final String customerRequestReference = "customer request reference 1";
 
         createIndividualRequest(customerRequestReference, LocalDateTime.now().plusDays((dataRetentionPeriod + 1) * -1));
-
-        final IIndividualRequest individualRequest =
-                individualRequestDao.getIndividualRequest(bulkCustomer, customerRequestReference, dataRetentionPeriod);
-
+        IIndividualRequest individualRequest = individualRequestDao.getIndividualRequest(bulkCustomer, customerRequestReference, dataRetentionPeriod);;
```

Comment:
Duplicated semi-colon at end of the line.

Reply:
removed